### ISSUE TITLE
🐞 Color output problem on Windows

### ISSUE THREAD

**Issue by MarvinJWendt** [gofiber/fiber#136]
**Fiber version**
`1.5.0` 

Hi, this problem occurs if you use Windows CMD to run fiber:
![image](https://user-images.githubusercontent.com/31022056/74392150-836dc180-4e06-11ea-9f48-e5c9fb9d036d.png)

Full Windows color support could possibly be accomplished by using the "Windows Way" of the [fathi/color](https://github.com/fatih/color) package:
![image](https://user-images.githubusercontent.com/31022056/74392243-c5970300-4e06-11ea-94d1-c464cc605ebc.png)

This problem also occurs if you use inbuilt terminals (in e.g. IntelliJ IDEA):
![image](https://user-images.githubusercontent.com/31022056/74392351-1870ba80-4e07-11ea-8195-410218f13443.png)

**Comment by MarvinJWendt** [comment]
Should I try to solve that via [fathi/color](https://github.com/fatih/color)?

**Comment by Fenny** [comment]
@MarvinJWendt, If it's possible, we should try to fix it without a third party library. 
I do not have access to a `Windows` OS or VM, could you try to fix it without  [fathi/color](https://github.com/fatih/color)?

**Comment by MarvinJWendt** [comment]
This would be really hard, we would have to create a handler which knows what shell is used and on which operating system the shell is running. 
Windows colors are a real pain, because every shell treats them differently.

This is an example of how much source code is needed to create such a handler:
https://github.com/mattn/go-colorable

Even this scripts, which should contain the windows color codes, is not displayed right:
https://gist.github.com/mlocati/fdabcaeb8071d5c75a2d51712db24011

![image](https://user-images.githubusercontent.com/31022056/74445288-7092d580-4e76-11ea-81cc-d90d6389a495.png)

TL;DR: There is no simple fix to support Windows colors, as the codes vary from shell to shell.

**Comment by Fenny** [comment]
We are removing the ANSI banner in v2, closing this issue.
